defaults/main.yml

```diff
@@ -1694,3 +1704,67 @@ owncloud__php__dependent_pools:
 owncloud__unattended_upgrades__dependent_origins:
   - origin: 'site=download.owncloud.org'
     state: '{{ "present" if (owncloud__auto_security_updates_enabled | bool) else "absent" }}'
+
+owncloud__permission_type: 'dpkg'
+
+owncloud__statoverride:
+    - mode: 640
+      path:
+        - "/var/www/owncloud/config/config.php"
```

Comment:
Note that the default "first level" quote in DebOps is `'`. This makes it easy to copy stuff around when you need to quote in Jinja2 (which would then use the `"`). Can you change this to `'/var/www/owncloud/config/config.php'` together with the other occurrences?